install/kubernetes/Makefile.values

```diff
@@ -33,7 +39,6 @@ export CILIUM_ETCD_OPERATOR_VERSION:=v2.0.7
 export CILIUM_ETCD_OPERATOR_DIGEST:=sha256:04b8327f7f992693c2cb483b999041ed8f92efc8e14f2a5f3ab95574a65ea2dc
 export CILIUM_NODEINIT_REPO:=quay.io/cilium/startup-script
 export CILIUM_NODEINIT_VERSION:=d69851597ea019af980891a4628fb36b7880ec26
-export CILIUM_OPERATOR_BASE_REPO:=quay.io/cilium/operator
```

Comment:
Doesn't this break something? :thinking: 

I still see other usages in the tree:

```
$ git grep CILIUM_OPERATOR_BASE_REPO
install/kubernetes/Makefile.values:export CILIUM_OPERATOR_BASE_REPO:=quay.io/cilium/operator
install/kubernetes/cilium/values.yaml.tmpl:    repository: "${CILIUM_OPERATOR_BASE_REPO}"
```

Reply:
oh yeah `CILIUM_OPERATOR_BASE_REPO` still exists. i needed to move it to 2 places:

- https://github.com/cilium/cilium/blob/c42ae6435e2eb19337792e8c98b133a840fb62d0/install/kubernetes/Makefile.values#L15
- https://github.com/cilium/cilium/blob/c42ae6435e2eb19337792e8c98b133a840fb62d0/install/kubernetes/Makefile.values#L22

because the image path might be different between release and CI images. it just happened to work for cilium/cilium because we use quay.io/cilium for both release and CI images.